modules/ROOT/pages/app-dashboards.adoc

```diff
@@ -36,7 +36,7 @@ Built-in dashboards include various pages to view different types of charts:
 [%header,cols="1,4"]
 |===
 | Built-In Dashboards | Charts
-| *Overview* a|
+| <<overview-charts>> a|
```

Comment:
For "Inbound - Mule messages", take out "Inbound", and just use "Mule messages".